inference-engine/src/legacy_api/src/transformations/convert_opset1_to_legacy/convert_prior_to_ie_prior.cpp

```diff
@@ -291,6 +291,6 @@ void ngraph::pass::ConvertPriorBox::convert_prior_box_clustered() {
         return true;
     };
 
-    auto m = std::make_shared<ngraph::pattern::Matcher>(unsqueeze, "CPUFusion.ConvertPriorBoxClusteredToPriorBoxClusteredIE");
+    auto m = std::make_shared<ngraph::pattern::Matcher>(unsqueeze, "CPUFusion_ConvertPriorBoxClusteredToPriorBoxClusteredIE");
```

Comment:
With this name we will not remove anything in CC. Can we rename it?

Reply:
It is a legacy name.
I made this change in order to avoid the warning:
```
/openvino/build/openvino/conditional_compilation/conditional_compilation_gen.h:87:30: warning: ISO C99 requires whitespace after the macro name [-Wc99-extensions]
#define ngraph_pass_CPUFusion.ConvertPriorBoxClusteredToPriorBoxClusteredIE 1
```